### ISSUE TITLE
A11y_AzureDataStudio_New deployment: SQL Server 2019 Big Data Cluster_Screen Reader: Narrator is not announcing the current status of checkbox after changing it as checked/unchecked.

### ISSUE THREAD

**Issue by Gayathri1501** [microsoft/azuredatastudio#18568]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG1.3.1; #Win32; #AINotInScope; #Info & Relationships; #Narrator; #NVDA;

### Environment Details:
Windows Version :Windows 11 Enterprise
Application name :Azure Data studio

### Repro Steps:
1.Turn on  the Narrator. 
2.Launch Azure Data Studio.
3.Select the . Select the Connections icon (if not already selected) from left bar.
4. Click on the ellipses '...' and select "New Deployment".
5. In the Deployment options, select "SQL Server Big Data Cluster".
6. In Step-1 of Deployment pre-requisites , there is a Accept terms of use checkbox.
7.Verify whether that the narrator is announcing the current status of checkbox after changing it or not.

### Note:
Similar issue can be observed in throughout the application whether the checkboxes are there.

### Actual Result:
Narrator is not announcing the current status of checkbox after changing it as checked/unchecked.

### Expected Result:
Narrator should announced the current status of checkbox after changing it as checked/unchecked.

### Attachment:
**Narrator Attachment:**
**![A11y_AzureDataStudio_New deployment SQL server 2019 Big Data Cluster_Screen Reader](https://user-images.githubusercontent.com/99962579/155975822-ba831722-73ae-4191-baa3-0e1085283b0b.gif)**
**NVDA Attachment:**
**![A11y_Azuredatastudio_New deployment SQL server_NVDA](https://user-images.githubusercontent.com/99962579/155982077-87369cb6-83f2-4a1b-9dd8-d3839c6ac070.gif)**

**Comment by Charles-Gagnon** [comment]
Closing this as an issue with Narrator - the state is announced correctly in NVDA and I can repro this myself in Chrome.

**Comment by Gayathri1501** [comment]
# External bug ID: https://microsoft.visualstudio.com/OS/_workitems/edit/40438048
We will take action on this bug once the external bug is fixed and closed.
And also, please confirm that this bug should close this bug only by Truster tester.  Thanks in Advance.
